src/components/Table/GridTable.tsx

```diff
@@ -959,6 +966,8 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
           ...(isHeader && stickyHeader && Css.sticky.top(stickyOffset).z1.$),
           // If we're within a card, use its background color
           ...(currentCard && Css.bgColor(currentCard.bgColor).$),
+          // Add in colspan css if needed
+          ...(currentColspan > 1 ? Css.gc(`${columnIndex + 1} / span ${currentColspan}`).$ : {}),
```

Comment:
Just curious, I thought this could be just `span 3`, but we do need the `3 / span 3`?

Reply:
Yea, we might be able to get away with `grid-column: span 3`, but I guess I was trying to be a bit more explicit with the definition. `grid-column` is shorthand for `grid-column-start` and `grid-column-end`.
`grid-column: 2 / span 3` translates to:
```
grid-column-start: 2;
grid-column-end: span 3;
```
`grid-column: span 3` translates to:
```
grid-column-start: span 3;
grid-column-end: auto;
```
The big difference here really is just the `auto`, which should translate to a default of `span 1`. 
After all that, yes you are right! ha. We could just do `grid-column: span 3`. I just kind of figured I'd be more explicit, but doesn't seem necessary.

Reply:
(updated to just use `span {currentColspan}`